### ISSUE TITLE
GraphQL Subscriptions

### ISSUE THREAD

**Issue by jacque006** [ConsenSys/ethql#124]
Are GraphQL subscriptions (https://graphql.org/blog/subscriptions-in-graphql-and-relay/) available and if not are there any plans to add them?

**Comment by kshinn** [comment]
Yes! There are definitely plans to add them. More details to come soon!
